docs/cli/getting-started/why-terramate.md

```diff
@@ -0,0 +1 @@
+# Why Terramate CLI?
```

Comment:
AFAICS this file is not used or referenced?

Reply:
yes correct - will delete (and add as a next step). good catch thank you